### ISSUE TITLE
Using display: table with margin incorrectly flagged as issue

### ISSUE THREAD

**Issue by adbutterfield** [SonarQubeCommunity/sonar-css#152]
I have something like:
```
.my-class {
  display: table;
  margin: 0 auto 1em;
}
```
And it gets flagged as this issue:
_Properties that do not work with the "display" property should be removed_

It looks like it's supposed to match on _display: table-*_, but it's actually matching on _display: table*_ maybe?

**Comment by racodond** [comment]
Thanks a lot for your feedback @adbutterfield!
Indeed, the rule was matching 'table' instead of 'table-'.
This is now fixed in the master branch: https://github.com/SonarQubeCommunity/sonar-css/commit/b3a26d20ea23edc4a8b14b5b97576d7f0c14395c

**Comment by adbutterfield** [comment]
No problem!
Thanks for this plugin!
